packages/babel-plugin-transform-remove-unused-catch-binding/src/index.js

```diff
@@ -0,0 +1,44 @@
+import syntaxOptionalCatchBinding from "babel-plugin-syntax-optional-catch-binding";
+
+export default function(babel) {
+  const { types: t } = babel;
+  return {
+    inherits: syntaxOptionalCatchBinding,
+
+    visitor: {
+      CatchClause(path) {
+        if (path.node.param === null) {
+          return;
+        }
+        if (t.isObjectPattern(path.node.param)) {
```

Comment:
If it's a pattern, we can just skip this as too much work.

Reply:
So, I had previously checked if the param was an identifier and if not exited early--is this what you mean, simply exit early if param not an identifier? (Also: thank you for your code reviews!)

Reply:
Yup. We could check all the binding identifiers inside a pattern, but that feels like too much work since the usual pattern is just `try {} catch(e) {}`

Reply:
OK, done.  Happy to take on any more advice :-)